### PR TITLE
chore(docs): Update kfp-tekton v2 version in the docs

### PR DESCRIPTION
**Pull request by Tomcli**
**Which issue is resolved by this Pull Request:** 
Resolves #

**Description of your changes:**

**Environment tested:**

* Python Version (use `python --version`):
* Tekton Version (use `tkn version`):
* Kubernetes Version (use `kubectl version`):
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: guides/kfp_tekton_install.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ To install the standalone Kubeflow Pipelines with Openshift Pipelines, run the f
 
 1. Follow the [Kubeflow install instructions](https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow-on-iks/#kubeflow-installation)
    to install the entire Kubeflow stack with `kfp-tekton`.
-   Kubeflow `v1.8.0` uses Tekton `v0.47.3` and `kfp-tekton` `v2.0.1` or `v1.7.1`. <!-- TODO update-->
+   Kubeflow `v1.8.0` uses Tekton `v0.47.3` and `kfp-tekton` `v2.0.3` or `v1.7.1`. <!-- TODO update-->
```
Comment (yhwang) [inline guides/kfp_tekton_install.md:182]:
```suggestion
   Kubeflow `v1.8.0` uses Tekton `v0.47.5` and `kfp-tekton` `v2.0.3` or `v1.7.1`. <!-- TODO update-->
```

Comment (Tomcli) [reply guides/kfp_tekton_install.md]: Thanks @yhwang

**Review by yhwang — APPROVED** [review]
/lgtm

**Pull request merged**